ppx-lib/gen_js_api_ppx.ml

```diff
@@ -44,6 +44,14 @@ let check_attribute = ref true
 let mark_as_handled_manually = ref (fun (_ : Parsetree.attribute) -> ())
 let used_attributes_tbl = Hashtbl.create 16
 
+(* [merlin_hide] tells merlin to not look at a node, or at any of its
+   descendants.  *)
+let merlin_hide =
+  { attr_name = { txt = "merlin.hide"; loc = Location.none }
+  ; attr_payload = PStr []
+  ; attr_loc = Location.none
+  }
```

Comment:
If you are linking with ppxlib, then I believe there is a `Ppxlib.Merlin_helpers` module that you could use, which already defines this attribute.

Reply:
Ppxlib is not a dependency here (it's only in the driver)